x11-themes/lxqt-themes/lxqt-themes-1.0.0.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="LXQt Themes"
+HOMEPAGE="https://lxqt-project.org/"
+
+if [[ ${PV} = *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/lxqt/${PN}.git"
+else
+	SRC_URI="https://github.com/lxqt/${PN}/releases/download/${PV}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~x86"
+fi
+
+LICENSE="LGPL-2.1+"
+SLOT="0"
+
+BDEPEND=">=dev-util/lxqt-build-tools-0.10.0"
+
+pkg_postinst() {
+	xdg_icon_cache_update
```

Comment:
Not inheriting xdg(-utils) though?

Reply:
If it emerges without issues as-is doesn't it mean the relevant eclass is inherited in some way? I can add it explicitly anyway

Reply:
Yeah, the 'xdg-utils' is inherited in the 'cmake.eclass', which is why it works, but if you were to run something like 'pkgcheck scan' on it you'd get a warning like so:

``` bash
x11-themes/lxqt-themes
  IndirectInherits: version 9999: xdg-utils: indirect inherit usage: 'xdg_icon_cache_update', line 25
```

How dangerous exactly that might be in the wild, I'm not sure of.  :]